.github/workflows/run_tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.9', '3.10', '3.11', '3.x' ]
+        python-version: [ '3.9', '3.10', '3.11' ]
```

Comment:
This is a temporary measure. `distutils` was deprecated and is no longer available in Python 3.12 (which was released recently). Somewhere in our dependency tree, something is trying to install `distutils`, so this is  causing issues. I think resolving that (requiring some sleuthing on our dependency tree) is outside the scope of this PR. Will address separately.